wagtailio/areweheadlessyet/management/commands/fetch_headless_issues.py

```diff
@@ -0,0 +1,64 @@
+import requests
+from django.core.management.base import BaseCommand
+
+from wagtailio.areweheadlessyet.models import WagtailHeadlessIssue
+
+
+class Command(BaseCommand):
+    help = "Fetch issues from Wagtail's repository tagged #headless."
+
+    def handle(self, *args, **options):
+        response = requests.get(
+            "https://api.github.com/repos/wagtail/wagtail/issues",
+            params={"labels": "headless"},
+        )
+
+        if not response.ok:
+            message = response.json()["message"]
+            self.stdout.write(
+                self.style.ERROR(
+                    f"Fetching from Github's API failed for the following reason: {message}"
+                )
+            )
+            return
```

Comment:
```suggestion
        response.raise_for_status()
```

Suggestion: We can simplify this down quite a lot, and just raise an exception instead. This has the added benefit of reporting back into Sentry in the event of an issue, rather than just the logs. Along with said exception, it also contains a bunch of context about the response and error, which would be useful for debugging.

Reply:
Alright, it's added.